src/realm/util/interprocess_condvar.cpp

```diff
@@ -218,8 +218,12 @@ void InterprocessCondVar::set_shared_part(SharedPart& shared_part, std::string b
     // terminates, the objects are destructed automatically by the kernel, so there will be no handle 
     // leaks or other kinds of leak.
 
-    std::string sem = "realm_sema_" + base_path + condvar_name;
-    std::string eve = "realm_event_" + base_path + condvar_name;
+    // replace backslashes because they're significant in object namespace names
```

Comment:
According to msdn, the problem was that backslashes are not allowed in kernel object names, except the backslash following "Local". You could update the comment with that